### PR TITLE
Add basic rebar.config, build pipeline, .gitignore

### PR DESCRIPTION
**Pull request by robertoaloi**

### CODE REVIEW

**Comment by potatosalad** [comment]
@robertoaloi Could we also [add metadata](https://hex.pm/docs/rebar3_publish#adding-metadata-to-code-classinlinesrcltmyappgtappsrccode) so this could be published to hex.pm?

File: rebar.lock [hunk 1]
```diff
@@ -0,0 +1 @@
+[].
```
Comment (huiliu2022) [inline rebar.lock:1]: is this file really needed?

**Comment by robertoaloi** [comment]
> @robertoaloi Could we also [add metadata](https://hex.pm/docs/rebar3_publish#adding-metadata-to-code-classinlinesrcltmyappgtappsrccode) so this could be published to hex.pm?

I was planning to add Hex support in a separate PR, but I can include it here if you prefer.

File: rebar.lock (hunk 1 above)
Comment (robertoaloi) [reply rebar.lock:1]: According to [rebar3's conventions](https://adoptingerlang.org/docs/development/dependencies/#dependency-lifecycles) the lock file should be kept version control, even if there are different opinions on this topic.
In this case the app doesn't have dependencies, so it's not a big deal. If we don't want to version control it, we should add it to the `.gitignore`, though.

**Comment by potatosalad** [comment]
> > @robertoaloi Could we also [add metadata](https://hex.pm/docs/rebar3_publish#adding-metadata-to-code-classinlinesrcltmyappgtappsrccode) so this could be published to hex.pm?
> 
> I was planning to add Hex support in a separate PR, but I can include it here if you prefer.

Separate PR also works, no preferences here 🙂
